src/server/BUILD.bazel

```diff
@@ -1,12 +1,37 @@
 load("//bazel/docker:defaults.bzl", "container_image", "container_layer")
+load("//bazel/js:defaults.bzl", "js_library", "nodejs_binary", "pkg_web")
+
+js_library(
+    name = "server",
+    srcs = [
+        ":getPageTemplate.js",
+        ":server.js",
+    ],
+    deps = [
+        "//src/client/routes:__pkg__",
```

Comment:
if this is why you need the other thing public, better to just state explicit dependency then expose it via public. Though it does sort of concern me that the server needs to rely on the routes to work, but maybe that's necessary.

Reply:
note, in this structure, only the dev server needs the routes to work, though these can be decoupled. Is it acceptable practice to run separate `ibazel` instances for two different parts of an app at the same time? like:

```bash
# terminal 1
./ibazel run //src/routes/secondary:dev

# terminal 2
./ibazel run //src/server:dev
```

Right now, only the latter is needed to run dev, and the docker image does not depend on routes

Reply:
That seems less than ideal? I think this is over optimizing at the moment, and for starters it's fine if we make the route config something that ends up in the docker container as part of the final product. But I expect that the dev server shouldn't even need the container. It's just a node server that looks somewhere in Bazel out for built artifacts and listens to the signals ibazel sends to know when to refresh the page.